convert/gitlab/convert.go

```diff
@@ -230,3 +212,50 @@ func (d *Converter) convert(ctx *context) ([]byte, error) {
 
 	return out, nil
 }
+
+func convertJobToStep(ctx *context, stageName string, jobname string, job *gitlab.Job) []*harness.Step {
+	var steps []*harness.Step
+	spec := new(harness.StepExec)
+
+	if job.Image != nil {
+		spec.Image = job.Image.Name
+		spec.Pull = job.Image.PullPolicy
+	} else if ctx.config.Default != nil && ctx.config.Default.Image != nil {
+		spec.Image = ctx.config.Default.Image.Name
+		spec.Pull = ctx.config.Default.Image.PullPolicy
+	} else if ctx.config.Image != nil {
+		spec.Image = ctx.config.Image.Name
+		spec.Pull = ctx.config.Image.PullPolicy
+	}
+
+	// Convert all scripts into a single step
+	script := append(job.Before)
+	script = append(script, job.Script...)
+	script = append(script, job.After...)
+	spec.Run = strings.Join(script, "\n")
+
+	steps = append(steps, &harness.Step{
+		Name: fmt.Sprintf("%s", jobname),
```

Comment:
since `jobname` is already a string, `fmt.Sprintf` should not be needed

Reply:
fixed.